app/src/main/java/com/teamagam/gimelgimel/app/injectors/modules/ViewModelModule.java

```diff
@@ -0,0 +1,62 @@
+package com.teamagam.gimelgimel.app.injectors.modules;
+
+import android.app.Activity;
+import android.content.Context;
+
+import com.teamagam.gimelgimel.app.common.launcher.Navigator;
+import com.teamagam.gimelgimel.app.injectors.scopes.PerActivity;
+import com.teamagam.gimelgimel.app.map.viewModel.adapters.GeoEntityTransformer;
+import com.teamagam.gimelgimel.app.message.viewModel.GeoMessageDetailViewModel;
+import com.teamagam.gimelgimel.app.message.viewModel.ImageMessageDetailViewModel;
+import com.teamagam.gimelgimel.app.message.viewModel.TextMessageDetailViewModel;
+import com.teamagam.gimelgimel.app.message.viewModel.adapter.MessageAppMapper;
+import com.teamagam.gimelgimel.domain.map.DrawMessageOnMapInteractorFactory;
+import com.teamagam.gimelgimel.domain.map.GoToLocationMapInteractorFactory;
+import com.teamagam.gimelgimel.domain.messages.DisplaySelectedMessageInteractorFactory;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+@PerActivity
+public class ViewModelModule {
```

Comment:
All this module can be discarded if we'd use @Inject and @PerActivity annotation in the viewmodel classes